mllib/src/main/scala/org/apache/spark/ml/r/GaussianMixtureWrapper.scala

```diff
@@ -91,7 +92,10 @@ private[r] object GaussianMixtureWrapper extends MLReadable[GaussianMixtureWrapp
       .setStages(Array(rFormulaModel, gm))
       .fit(data)
 
-    new GaussianMixtureWrapper(pipeline, dim)
+    val gmm: GaussianMixtureModel = pipeline.stages(1).asInstanceOf[GaussianMixtureModel]
```

Comment:
they are the same when the pipeline has 1 stage.
I prefer `stages.last` because if we later add a stage to transform the input data it will break `stages(1)`